include/net/ethernet.h

```diff
@@ -27,6 +27,7 @@
 #define ETH_FCS_LEN			4			/* Octets in the FCS */
 
 /* Ethernet protocol ID's */
+#define ETHERTYPE_ALL		0x0003		/* Every packet (be careful!!!) */
```

Comment:
uh, I see that there are some styling issues with the file - but let's not change it in this PR.